README.md

```diff
@@ -12,6 +12,13 @@ bin/console plugin:refresh
 bin/console plugin:install -a iMidiCategoryDuplicator
 ```
 
+## Version Support
+
+* For Shopware 6.4 use the latest 1.x Version
+* For Shopware 6.6 use the latest 3.x Version
+
+We currently do not plan support for Shopware 6.5 - we spare out the 2.x versions, in case anyone wants to add support. 
+
```

Comment:
Not entirely true… the (previously) used `~6.4` includes all versions < 7.0.0, see [semver.madewithlove.com](https://semver.madewithlove.com/?package=shopware%2Fcore&constraint=~6.4) to believe. 😎

You probably would have wanted to use `~6.4.0` (and for the new release `~6.6.0`) to make it work for the versioning policy of Shopware (where minor versions are in fact major versions)…

Reply:
true :-)